src/leap/keymanager/__init__.py

```diff
@@ -232,6 +234,40 @@ def _get_key_from_nicknym(self, address):
         #     'Content-type is not JSON.')
         defer.returnValue(json_content)
 
+    def _key_search_failed(self, code):
+        """
+        Checks if response code is NOT_FOUND.
+
+        :param code: The response code, after trying to fecth a key from nicknym.
+        :type code: int
+
+        :return: boolean
+        """
+        return code == NOT_FOUND
```

Comment:
I don't think this method is needed. The docstring is way longer than the code. I think it's easier to read if this code lives directly in the if below.